src/cpp/src/continuous_batching_impl.hpp

```diff
@@ -89,6 +88,10 @@ class ContinuousBatchingPipeline::ContinuousBatchingImpl : public ContinuousBatc
                            const ov::AnyMap& properties,
                            const ov::genai::GenerationConfig& generation_config,
                            bool is_validation_mode_enabled = false);
+    
+    virtual ~ContinuousBatchingImpl() {
```

Comment:
move to .cpp ? all other methods have only declaration in this file

Reply:
Done